Excalidash - update script fails.

### ✅ Have you read and understood the above guidelines?

Yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

excalidash

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/excalidash.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 🧱 Is this Proxmox host running arm64?

No

### 📈 Which Proxmox version are you on?

9.2.11

### 📝 Provide a clear and concise description of the issue.

When running "update" with the goal of upgrading excalidash from 0.5.1 to version 0.6.0 , the process fails and leaves the LXC in a non-operational state.

### 🔄 Steps to reproduce the issue.

Run the **update** command as described above.

### ❌ Paste the full error output (if available).

<.... slew of files not found......>
find: ‘/opt/excalidash/frontend/node_modules/detect-node-es’: No such file or directory
find: ‘/opt/excalidash/frontend/node_modules/bidi-js’: No such file or directory
find: ‘/opt/excalidash/frontend/node_modules/is-extglob’: No such file or directory
find: ‘/opt/excalidash/frontend/node_modules/png-chunk-text’: No such file or directory
find: ‘/opt/excalidash/frontend/node_modules/min-indent’: No such file or directory
find: ‘/opt/excalidash/frontend/node_modules/postcss-js’: No such file or directory
find: ‘/opt/excalidash/frontend/dist’: No such file or directory
find: ‘/opt/excalidash/frontend/scripts’: No such file or directory
  ✔️  Deployed: excalidash (0.6.0)
  ✔️  Configured Database Provider

added 350 packages, and audited 351 packages in 2m

67 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
npm notice
npm notice New major version of npm available! 10.8.2 -> 12.0.2
npm notice Changelog: https://github.com/npm/cli/releases/tag/v12.0.2
npm notice To update run: npm install -g npm@12.0.2
npm notice
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma

✔ Generated Prisma Client (v5.22.0) to ./src/generated/client in 303ms

Start by importing your Prisma Client (See: https://pris.ly/d/importing-client)

Tip: Want to turn off tips and other hints? https://pris.ly/tip-4-nohints

Need to install the following packages:
tsc@2.0.4
Ok to proceed? (y) y

npm warn deprecated tsc@2.0.4: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.

                                                                               
                This is not the tsc command you are looking for                
                                                                               

To get access to the TypeScript compiler, tsc, from the command line either:

- Use npm install typescript to first add TypeScript to your project before using npx
- Use yarn to avoid accidentally running code from un-installed packages
  ✖️  in line 52: exit code 1 (General error / Operation not permitted): while executing command  npx tsc

  --- Last 20 lines of log ---
[2026-08-24 08:37:40] [INFO] Checking for update: excalidash
[2026-08-24 08:37:41] [OK] Update available: excalidash 0.5.1 → 0.6.0
[2026-08-24 08:37:41] [INFO] Stopping Service
[2026-08-24 08:37:41] [OK] Stopped Service
[2026-08-24 08:37:41] [INFO] Fetching GitHub release: excalidash (0.6.0)
[2026-08-24 08:37:46] [OK] Deployed: excalidash (0.6.0)
[2026-08-24 08:37:46] [INFO] Configuring Database Provider (postgresql)
[2026-08-24 08:37:46] [OK] Configured Database Provider
[2026-08-24 08:37:46] [INFO] Rebuilding Application
[2026-08-24 08:39:24] [ERROR] in line 52: exit code 1 (General error / Operation not permitted): while executing command  npx tsc
  -----------------------------------

### 🖼️ Additional context (optional).

_No response_

--- a/ct/excalidash.sh
+++ b/ct/excalidash.sh
@@ -47,11 +47,11 @@ function update_script() {
     msg_ok "Configured Database Provider"
 
     msg_info "Rebuilding Application"
-    $STD npm ci
+    $STD npm ci --include=dev
     $STD npx prisma generate
     $STD npx tsc
     cd /opt/excalidash/frontend
-    $STD npm ci
+    $STD npm ci --include=dev
     $STD npm run build
     cp -r /opt/excalidash/frontend/dist/. /var/www/excalidash/
     msg_ok "Rebuilt Application"

